minitests/litex/uart_ddr/arty/README.md

```diff
@@ -0,0 +1,90 @@
+# LiteX UART DDR minitest
```

Comment:
Please add info about the exact versions of Litex repos (Litex and Litedram) used to generate this minitest

Reply:
Done